dataflow-migrate-schedules/src/main/java/io/spring/migrateschedule/service/DockerImage.java

```diff
@@ -0,0 +1,212 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *          http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package io.spring.migrateschedule.service;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.regex.Pattern;
+
+/**
+ * Docker image name parsing utility Code from https://github.com/vmware/admiral
+ */
+public class DockerImage {
```

Comment:
Not sure I understand how is this class used.

Reply:
Good Catch.  During my experiment to try and find data this artifact was required.   Now that the migrator is getting the data directly from the manifest, that code was removed and this is no longer needed.